docs/section22/Rule22-17.md

```diff
@@ -0,0 +1,52 @@
+
+# CHW&CW - Rule 22-17  
+
+**Schema Version:** 0.0.25    
+**Mandatory Rule:** True  
+**Rule ID:** 22-17  
+**Rule Description:** The baseline heat rejection device shall have an efficiency of 38.2 gpm/hp  
+**Rule Assertion:** B-RMR = expected value  
+**Appendix G Section:** Section 22 CHW&CW Loop  
+**90.1 Section Reference:** G3.1.3.11
+**Data Lookup:** None  
+**Evaluation Context:** Each HeatRejection 
+**Applicability Checks:**  
+
+1. B-RMR is modeled with at least one air-side system that is Type-7, 8, 12, 13, 7b, 8b, 12b.
+2. B-RMR is modeled with heat rejection loop
+
+**Function Call:**  
+
+1. get_heat_rejection_loops_connected_to_baseline_systems()
+
+**Applicability Checks:**  
+
+- The function get_heat_rejection_loops_connected_to_baseline_systems() returns a list of loops that are connected to Type 7,8,12,13,7b,8b,12b.  Get the list of applicable heat rejection loops: `heat_rejection_loop_ids = get_heat_rejection_loops_connected_to_baseline_systems(B_RMI)`
+
+- check applicability for each HeatRejection in the model: `for heat_rejection in B_RMI.heat_rejections:`
+
+  - check if heat_rejection fluid_loop is one of the applicable loops, if so, continue to the rule logic: `if heat_rejection.loop.id in heat_rejection_loop_ids: CONTINUE TO RULE LOGIC`
+  
+  - Else, rule is not applicable to the heat rejection: `else: RULE_NOT_APPLICABLE`
+
+  ## Rule Logic:  
+
+  - get the fan_electric_input_power for the heat rjection: `fan_W = heat_rejection.fan_nameplate_power`
+  
+  - get the design water flow rate (assumed to be gpm): `design_water_flowrate = heat_rejection.design_water_flowrate`
+  
+  - calculate the efficiency in gpm/hp, the following equation includes a conversion from Watts to hp: `heat_rejection_efficiency = design_water_flowrate / (fan_W * 0.00134102)`
+
+  **Rule Assertion - HeatRejection:**
+
+  - Case 1: If the heat_rejection_efficiency equals 38.2 gpm/hp, PASS: `if ( heat_rejection_efficiency == 38.2 ): PASS`
+
```

Comment:
@mkarpman do we need to include a conservative outcome here?

Reply:
I think so. If the value is higher, we would consider it conservative.

Reply:
is this the ahj_compare flag?

Reply:
@KarenWGard and @mkarpman - why do we need 3 cases here. Can it simply be one case, which always results in an undetermined outcome and the message simple states 
The project includes a cooling tower.  We calculated the cooling tower efficiency to be " + heat_rejection_efficiency + ", and the required efficiency is 38.2 gpm / hp.  However, it was not possible to verify that the modeling inputs correspond to the rating conditions in Table 6.8.1-7.  Please review the efficiency and ensure that it is correct at the rating conditions as specified in the Table 6.8.1-7." + addtitional_note_for_no_shaft_power`

Reply:
I don't have strong feelings one way or the other. I thought it may be worth mentioning that our calculation includes rule of thumb hp to bhp conversion in cases when fan_shaft_power is not entered, but I may be overthinking it.